### DISCUSSION TITLE
工科必须用电脑吗

### DISCUSSION THREAD

**Discussion by QingHuaniU6** [CollegesChat/university-information#238] [Q&A]
工科必须用电脑吗，电脑是必需品吗，可不可以带台式主机。

**Comment by SummerBeachhh** [comment]
工业设计软件，编程软件，哪个不需要电脑？台式主机不会触发宿舍限电你可以带，但是如果老师让你们带着电脑去实训之类的，不会拎着主机去吧？

**Comment by gydchenxiao** [comment]
工科肯定得用电脑啊，用visual code vs code,idea,Qt，Android studio等编译器做web，小程序，app,桌面应用程序，嵌入式得用keil5，去实验室还是带个笔记本电脑方便一些。

**Comment by pupupu147** [comment]
工科很多课程都需要用到电脑的，上大学其实都需要，但是工科对电脑要求还要高一点，得带得动程序运行

**Comment by WoooD-FuZao** [comment]
工科就得用电脑，我不知道你是什么专业的我本人是机械，反正我平时做建模的时候必须用电脑，而且你在做很多事情的时候都要用电脑
